opt/prepare-aiidalab.sh

```diff
@@ -64,15 +64,16 @@ if [[ ${INITIAL_SETUP} == true ||  "${AIIDALAB_SETUP}" == "true" ]]; then
   if [ ! -e /home/${SYSTEM_USER}/apps/aiidalab-widgets-base ]; then
     git clone https://github.com/aiidalab/aiidalab-widgets-base /home/${SYSTEM_USER}/apps/aiidalab-widgets-base
     cd /home/${SYSTEM_USER}/apps/aiidalab-widgets-base
-    git checkout ${AIIDALAB_DEFAULT_GIT_BRANCH}
-    git reset --hard v1.0.0b18
+    git checkout ${AWB_APP_VERSION}
```

Comment:
```suggestion
    git checkout "${AWB_APP_VERSION}"
```

Reply:
Just realized, this missing the dependency installation, is it not?

Reply:
> Just realized, this is missing the dependency installation, is it not?

That depends on whether we want to keep the `aiidalab-widgets-base` **package** linked to the image version.

If we link it - then it is listed as the dependency in the `requirements.txt` file. If we don't - then we install the dependencies here.

Reply:
I'm not sure what you mean by this. The AWB version is now "linked" through the AWB_APP_VERSION env variable, so it should be installed here.

Reply:
Ok, I think I understand better now what you mean, but that would be very non-intuitive if we install a different package version somewhere else, don't you think?

Reply:
Agreed: I added the pip install.